docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,191 @@
+
+    if (recurrence == null) {
+        console.log("Non-recurring meeting");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following example shows the results of the `getAsync` call that retrieves the recurrence for a series.
+
+> [!NOTE]
+> In this example, `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```json
+{
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get recurrence as an attendee
+
+In the following example, an appointment attendee can get the recurrence object of an appointment series given the series, an instance of that series, or a meeting request.
+
+```js
+outputRecurrence(Office.context.mailbox.item);
+
+function outputRecurrence(item) {
+    var recurrence = item.recurrence;
+    var seriesId = item.seriesId;
+
+    if (recurrence == null) {
+        console.log("Non-recurring item");
+        return;
+    }
+
+    console.log(JSON.stringify(recurrence));
+}
+```
+
+The following example shows the value of the `item.recurrence` property for an appointment series.
+
+> [!NOTE]
+> In this example, `seriesTimeObject` is a placeholder for the JSON representing the `recurrence.seriesTime` property. You should use the [`SeriesTime`][SeriesTime link] methods to get the recurrence date and time properties.
+
+```json
+{
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {
+        "interval": 1,
+        "days": ["tue","thu"],
+        "firstDayOfWeek": "sun"},
+    "seriesTime": {seriesTimeObject},
+    "recurrenceTimeZone": {
+        "name": "Pacific Standard Time",
+        "offset": -480}}
+```
+
+### Get the recurrence details
+
+After you've retrieved the recurrence object (either from the `getAsync` callback or from `item.recurrence`), you can get specific properties of the recurrence. For example, you can get the start and end dates and times of the series by using [methods][SeriesTime link] on the `recurrence.seriesTime` property.
+
+```js
+// Get series date and time info
+var seriesTime = recurrence.seriesTime;
+var startTime = recurrence.seriesTime.getStartTime();
+var endTime = recurrence.seriesTime.getEndTime();
+var startDate = recurrence.seriesTime.getStartDate();
+var endDate = recurrence.seriesTime.getEndDate();
+var duration = recurrence.seriesTime.getDuration();
+
+// Get series time zone
+var timeZone = recurrence.recurrenceTimeZone;
+
+// Get recurrence properties
+var recurrenceProperties = recurrence.recurrenceProperties;
+
+// Get recurrence type
+var recurrenceType = recurrence.recurrenceType;
+```
+
+## See also
+
```

Comment:
Are these links formatted correctly?  First item in list is missing the colon like the other items in the list. Shouldn't links be formatted like this? 
`[getAsync](/javascript/api/outlook_1_7/office.recurrence#getasync-options--callback-)`